src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -651,6 +695,35 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(
   return S_OK;
 }
 
+HRESULT STDMETHODCALLTYPE CorProfiler::JITInlining(FunctionID callerId, 
+    FunctionID calleeId, BOOL* pfShouldInline) {
+  ModuleID calleeModuleId;
+  mdToken calleFunctionToken = mdTokenNil;
+  auto hr = this->info_->GetFunctionInfo(calleeId, NULL, &calleeModuleId,
+                                         &calleFunctionToken);
+
+  *pfShouldInline = true;
+
+  if (FAILED(hr)) {
+    Warn("*** JITInlining: Failed to get the function info of the calleId: ", calleeId);
+    return S_OK;
+  }
+
+  RejitHandlerModule* handlerModule = nullptr;
+  if (rejit_handler->TryGetModule(calleeModuleId, &handlerModule)) {
+    RejitHandlerModuleMethod* handlerMethod = nullptr;
+    if (handlerModule->TryGetMethod(calleFunctionToken, &handlerMethod)) {
+      Info("*** JITInlining: Inlining disabled for:  moduleId: ", calleeModuleId,
```

Comment:
Did you want to keep this around as a Debug log? Also, at the end of the message you added a `]` character. Are you missing a `[` character somewhere?

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/9e7bcb3955022de47db4e80e29c33db9f64b69c2